pit30m/data/log_reader.py

```diff
@@ -225,16 +273,14 @@ def stereo_calib(self):
             return data
 
     @cached_property
-    def raw_pose_data(self) -> np.ndarray:
-        """Returns the raw pose array, which needs manual association with other data types. 100Hz.
+    def raw_pose_data(self) -> npt.NDArray[RAW_POSE_DTYPE]:
```

Comment:
omg I didn't know you could do that! so cool! 😁

Reply:
Yep, just gotta actually run `mypy` now :P